README.md

```diff
@@ -41,3 +41,19 @@ To run a single script, you can mount it in a volume under `/usr/src/app`. From
 ```
 $ docker run -v ${PWD}:/usr/src/app -w /usr/src/app -it --rm iojs iojs index.js
 ```
+
+## Image Varients
+
+The `iojs` images come in three flavors, each designed for a specific use case. 
+
+`<version>`:
+
+This is the defacto image. If you are unsure about what your needs are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code and start the container to start your app), as well as the base to build other images off of.
+
+`<version>-onbuild`:
+
+This image makes building derivitative images easier. For most use cases, creating a `Dockerfile` in the base of your project directory with the line `FROM iojs:onbuild` will be enough to create a stand-alone image for your project.
```

Comment:
I think it would be good to use something closer to the in https://github.com/iojs/docker-iojs/pull/31#issuecomment-75665732 for this tag, as this one doesn't seem to make it entirely obvious that it'll run an app automatically. Pasting the description I meant below:
"These images include all the packages from the <version>-tagged docker build, but will also install dependencies specified in a package.json file with npm and run npm start in the directory /usr/src/app, which it creates inside the container and copies a local directory you specify to."